packages/hooks/src/useCurrencyConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { useFetch } from '@deriv/api';
+
+const useCurrencyConfig = (currency: string) => {
+    const { data, ...rest } = useFetch('website_status');
+    const currency_config = data?.website_status?.currencies_config?.[currency];
+    const modified_currency_config = currency_config
+        ? {
+              ...currency_config,
+              is_crypto: currency_config.type === 'crypto',
+              is_fiat: currency_config.type === 'fiat',
+              is_aud: currency === 'AUD',
+              is_usd: currency === 'USD',
+              is_eur: currency === 'EUR',
+              is_gbp: currency === 'GBP',
+              is_eth: currency === 'ETH',
+              is_btc: currency === 'BTC',
+              is_ltc: currency === 'LTC',
+              is_usdt: currency === 'USDT',
```

Comment:
not important but as I checked we don't have this in the response :thinking:

Reply:
@shayan-deriv Yeah you are right, I guess this is the same thing I raised [here](https://github.com/binary-com/deriv-app/pull/8772#discussion_r1218803483) 🤔 I think we should map `UST` to `USDT` but I need someone to confirm it 😅

@nijil-deriv Any idea?

Reply:
As checked with @nijil-deriv, We should map `UST` to `USDT`.